rbi/sorbet/builder.rbi

```diff
@@ -3,20 +3,37 @@ class T::Private::Methods::DeclBuilder
   sig {params(params: T.untyped).returns(T::Private::Methods::DeclBuilder)}
   def type_parameters(*params); end
 
-  sig {returns(T::Private::Methods::DeclBuilder)}
-  def abstract; end
-
-  sig {returns(T::Private::Methods::DeclBuilder)}
-  def final; end
+  sig do
+    params(
+      blk: T.nilable(T.proc.bind(T::Private::Methods::DeclBuilder).void)
```

Comment:
If these methods require a block (which we have tests for in other places), why is the block argument marked as nilable?  (Other methods in this RBI have the same issue.)  Are we trying to avoid double errors, one from the rewriter pass, and one from the typechecking pass?  If that's the only reason, I think we should just accept the duplicate errors and make these signatures correct by removing `T.nilable`.